python_app/app.py

```
@@ -0,0 +1,64 @@
+import hashlib
+import datetime
+from typing import Any, Dict, List, Optional, Tuple
+from pathlib import Path
+
+from flask import Flask, render_template, jsonify, request, redirect, url_for
+
+from models import SensorData, ConnectionState, get_mock_sensor_data, get_mock_connection_state
+from utils import format_reading, parse_sensor_payload, validate_beacon_id
+
+app = Flask(__name__)
+app.config["SECRET_KEY"] = "dev-secret-key"
```

Ruff (0.15.6)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)